view/templates/manifest.tpl

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "{{$title}}",
+  "start_url": "{{$baseurl}}",
+  "display": "standalone",
+  "description": "A Decentralized Social Network",
```

Comment:
Do you mean using the sitename as the description? They are two different things (although I don't really see what Android uses the description for).